testers.json

```diff
@@ -3565,217 +3565,217 @@
+    },
+    "Stage 2›throw expressions›logical": {
+      "spec": "https://github.com/tc39/proposal-throw-expressions",
+      "code": "var a, b;\ntry {\n  a = 19 || throw 77;\n  b = 88 && throw 23;\n} catch (e) {\n  return a + e === 42;\n}"
+    },
+    "Stage 2›throw expressions›parameter initializers": {
+      "spec": "https://github.com/tc39/proposal-throw-expressions",
+      "code": "function fn (arg = throw 42) {\n  return arg;\n}\n\nif (fn(21) !== 21) return false;\n\ntry {\n  fn();\n} catch (e) {\n  return e === 42;\n}"
+    },
+    "Stage 2›throw expressions›arrow function bodies": {
+      "spec": "https://github.com/tc39/proposal-throw-expressions",
+      "code": "var fn = () => throw 42;\ntry {\n  fn();\n} catch (e) {\n  return e === 42;\n}"
+    },
+    "Stage 2›throw expressions›conditionals": {
+      "spec": "https://github.com/tc39/proposal-throw-expressions",
+      "code": "true ? 42 : throw 21;\ntry {\n  false ? 42 : throw 21;\n} catch (e) {\n  return e === 21;\n}"
+    },
+    "Stage 2›Set methods›Set.prototype.intersection": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "var set = new Set([1, 2, 3]).intersection(new Set([2, 3, 4]));\nreturn set.size === 2\n  && set.has(2)\n  && set.has(3);"
+    },
+    "Stage 2›Set methods›Set.prototype.union": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "var set = new Set([1, 2]).union(new Set([2, 3]));\nreturn set.size === 3\n  && set.has(1)\n  && set.has(2)\n  && set.has(3);"
+    },
+    "Stage 2›Set methods›Set.prototype.difference": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "var set = new Set([1, 2, 3]).difference(new Set([3, 4]));\nreturn set.size === 2\n  && set.has(1)\n  && set.has(2);"
+    },
+    "Stage 2›Set methods›Set.prototype.symmetricDifference": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "var set = new Set([1, 2]).symmetricDifference(new Set([2, 3]));\nreturn set.size === 2\n  && set.has(1)\n  && set.has(3);"
+    },
+    "Stage 2›Set methods›Set.prototype.isDisjointFrom": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "return new Set([1, 2, 3]).isDisjointFrom([4, 5, 6]);"
+    },
+    "Stage 2›Set methods›Set.prototype.isSubsetOf": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "return new Set([1, 2, 3]).isSubsetOf([5, 4, 3, 2, 1]);"
+    },
+    "Stage 2›Set methods›Set.prototype.isSupersetOf": {
+      "spec": "https://github.com/tc39/proposal-set-methods",
+      "code": "return new Set([5, 4, 3, 2, 1]).isSupersetOf([1, 2, 3]);"
+    },
+    "Stage 2›ArrayBuffer.prototype.transfer›ArrayBuffer.prototype.transfer()": {
```

Comment:
transfer and transferToFixedLength and .disposed are stage 3 now (realloc is gone)